.vscode/launch.json

```diff
@@ -0,0 +1,38 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: nnvg",
+            "type": "python",
+            "request": "launch",
+            "program": "${workspaceFolder}/src/nnvg",
+            "cwd": "${workspaceFolder}/src",
+            "args": ["--templates", "${workspaceFolder}/test/gentest_dsdl/templates",
+                     "--generate-namespace-types",
+                     "--verbose",
+                     "--outdir", "${workspaceFolder}/.vscode/nunavut_out",
+                     "-l", "c",
+                     "${workspaceFolder}/test/gentest_dsdl/dsdl/uavcan"]
+        },
+        {
+            "name": "Pytest: current test",
+            "type": "python",
+            "request": "launch",
+            "module": "pytest",
+            "args": ["-k", "test_namespace_stropping"],
```

Comment:
Why is there a command specific to a particular test?

Reply:
crap. Good catch.